BrowserKit/Sources/ToolbarKit/ToolbarButton.swift

```diff
@@ -10,7 +10,7 @@ public enum ToolbarButtonGesture {
     case longPress
 }
 
-class ToolbarButton: UIButton, ThemeApplicable {
+public class ToolbarButton: UIButton, ThemeApplicable {
```

Comment:
I don't think we should make this button public. We should use a `UIButton` instead in the delegate methods.